<launch>
   <param name="use_sim_time" value="true" />
   <env name="ROSCONSOLE_CONFIG_FILE"  value="$(find zachet)/cfg/rosconsole.conf"/>
   <node pkg="stage_ros" type="stageros" name="model"
     args="$(find zachet)/worlds/task1.world"/>
   
  <node pkg="zachet" type="zachet" name="zachet" output="screen" >
	<remap from="scan" to="base_scan"/>
	<remap from="pose" to="base_pose_ground_truth"/>
	<param name="min_range" value="1" />
	<param name="max_v" value="1" />
	<param name="max_w" value="0.5" />
	<param name="angle" value="30" />
	<param name="k_omega" value="20" />
	<param name="k_vel" value="0.01" />
	
   </node>
</launch>
